### PR TITLE
(#57) Update NuGet package icon to cake-addin icon

### LINKED ISSUE #57: Update Cake.FileHelpers NuGet package icon to the new Cake-Contrib logo for addins
https://github.com/cake-contrib/graphics

image

### PR DESCRIPTION
**Pull request by augustoproiete**
Closes #57

### CODE REVIEW

File: src/Cake.FileHelpers/Cake.FileHelpers.csproj [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <Copyright>Copyright 2017-$([System.DateTime]::Now.Year) - Cake Contributions</Copyright>
     <PackageProjectUrl>https://github.com/cake-contrib/Cake.FileHelpers</PackageProjectUrl>
     <PackageIcon>icon.png</PackageIcon>
```
Comment (nils-a) [inline src/Cake.FileHelpers/Cake.FileHelpers.csproj:21]: You can remove this line, as well as the the `icon.png` file. `CakeContrib.Guidelines` adds those automatically, now.

File: src/Cake.FileHelpers/Cake.FileHelpers.csproj [hunk 2]
```diff
@@ -19,7 +19,7 @@
...
-    <PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/cake-contrib-medium.png</PackageIconUrl>
+    <PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/addin/cake-contrib-addin-medium.png</PackageIconUrl>
```
Comment (nils-a) [inline src/Cake.FileHelpers/Cake.FileHelpers.csproj:22]: You could remove this line. `CakeContrib.Guidelines` adds this automatically, now.

**Review by nils-a — APPROVED** [review]
While the change would work as intended, I left two comments 👼

File: src/Cake.FileHelpers/Cake.FileHelpers.csproj (hunk 1 above)
Comment (augustoproiete) [reply src/Cake.FileHelpers/Cake.FileHelpers.csproj]: Awesome. Removed!

**Review by nils-a — APPROVED** [review]
LGTM

**Pull request merged**